plugins/sonarqube-actions/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "@janus-idp/sonarqube-actions",
```

```suggestion
  "name": "@janus-idp/backstage-scaffolder-backend-module-sonarqube",
```

Looking at all available scaffolder plugins out there, all use a more descriptive name that contains `scaffolder-backend-module`. Let's follow the conventions, so the module is easier to discover for users.